bolt/src/main/java/com/slack/api/bolt/request/builtin/EventRequest.java

```diff
@@ -122,9 +122,12 @@ public EventRequest(
             JsonObject context = thread.get("context").getAsJsonObject();
             if (context != null) {
                 AssistantThreadContext threadContext = AssistantThreadContext.builder()
-                        .enterpriseId(context.get("enterprise_id") != null ? context.get("enterprise_id").getAsString() : null)
-                        .teamId(context.get("team_id") != null ? context.get("team_id").getAsString() : null)
-                        .channelId(context.get("channel_id") != null ? context.get("channel_id").getAsString() : null)
+                        // enterprise_id here can be a null value
```

Comment:
while testing this change, I found that this pattern could result in a runtime error. I don't think this was the same when we released this feature but the error needs to be eliminated anyway